### ISSUE TITLE
Micromamba search is not Displaying channel name. Its just displaying the URL in case of internal channels, trimming the name of channel

### ISSUE THREAD

**Issue by rmittal87** [mamba-org/mamba#2875]
### Troubleshooting docs

- [X] My problem is not solved in the Troubleshooting docs

### How did you install Mamba?

Micromamba

### Search tried in issue tracker

no issue

### Latest version of Mamba

- [X] My problem is not solved with the latest version

### Tried in Conda?

I do not have this problem with Conda, just with Mamba

### Describe your issue

Micromamba is not showing the channel name  from where the package is searched, This issue is introduced in 1.4.3v of micromamba  

### Logs

```shell
micromamba search xerces-c -c https://conda.abc.com/conda-forge 
Getting repodata from channels...


 Name     Version Build                    Channel
─────────────────────────────────────────────────────────────────────────────
 xerces-c 3.2.0   0                        conda.abc.com
 xerces-c 3.1.4   3                        conda.abc.com
 xerces-c 3.1.4   2                        conda.abc.com
 xerces-c 3.1.4   1                        conda.abc.com
 xerces-c 3.1.4   0                        conda.abc.com
```

**Comment by jonashaag** [comment]
Is this different in micromamba 1.4.2?

**Comment by rmittal87** [comment]
yes , in 1.4.2 , its clearly mentioning the channels

 xerces-c 3.2.0   0
conda.abc.com/files/channels/conda-forge/linux-64
 xerces-c 3.1.4   3
conda.abc.com/files/channels/conda-forge/linux-64
 xerces-c 3.1.4   2
conda.abc.com/files/channels/conda-forge/linux-64
 xerces-c 3.1.4   1
conda.abc.com/files/channels/conda-forge/linux-64
 xerces-c 3.1.4   0
conda.abc.com/files/channels/conda-forge/linux-64

**Comment by jjerphan** [comment]
Fixed in 2.0.

```
$ micromamba search python       
```

```
Getting repodata from channels...

conda-forge/noarch                                  17.6MB @   5.3MB/s  3.4s
conda-forge/linux-64                                40.4MB @   5.8MB/s  7.0s
 Name   Version   Build                                Channel     Subdir     
───────────────────────────────────────────────────────────────────────────────
 python 3.9.9     h543edf9_0_cpython     (+  1 builds) conda-forge conda-forge
 python 3.9.7     h49503c6_0_cpython     (+  4 builds) conda-forge conda-forge
 python 3.9.6     h49503c6_0_cpython     (+  1 builds) conda-forge conda-forge
 python 3.9.5     h49503c6_0_cpython                   conda-forge conda-forge
...
```


Feel free to reopen an issue with a reproduce if you still have this issue with 2.0.

Note that 1.x is note being developed anymore, but to address security fixes.
